### PR TITLE
Fix protemiqon-peptidespectrummatching build

### PR DESCRIPTION
**Pull request by ZimmerA**
This PR attempts to fix the build process of protemiqon-peptidespectrummatching using the approach taken in https://github.com/bioconda/bioconda-recipes/blob/master/recipes/flashlfq/meta.yaml.

### CODE REVIEW

File: recipes/proteomiqon-peptidespectrummatching/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-    - dotnet tool install --add-source $SRC_DIR --tool-path $PREFIX/bin --version {{ version }} ProteomIQon.PeptideSpectrumMatching
+  number: 1
 
 requirements:
-  host:
-    - dotnet-sdk
+  build:
   run:
-    - dotnet-runtime
+    - openssl =1.1
```
Comment (bgruening) [inline recipes/proteomiqon-peptidespectrummatching/meta.yaml:19]: is this pin necessary? It usually not good to bin such a critical library

Comment (ZimmerA) [reply recipes/proteomiqon-peptidespectrummatching/meta.yaml]: Lets see!

Comment (ZimmerA) [reply recipes/proteomiqon-peptidespectrummatching/meta.yaml]: https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=15370&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=1654 Looks like we need it? After all https://github.com/bioconda/bioconda-recipes/blob/master/recipes/flashlfq/meta.yaml probably included it for the same reasons...

Comment (ZimmerA) [reply recipes/proteomiqon-peptidespectrummatching/meta.yaml]: Aaah, you were asking about the pin... My bad 😊

Comment (ZimmerA) [reply recipes/proteomiqon-peptidespectrummatching/meta.yaml:20]: Looks like a hard requirement...

**Comment by ZimmerA** [comment]
@bgruening I think this is ready to be merged!

**Comment by ZimmerA** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**